src/DssSpell.t.base.sol

```diff
@@ -1140,6 +1238,317 @@ contract DssSpellTestBase is Config, DssTest {
+            ) = reg.info(p.ilk);
+            assertEq(name,     GemAbstract(p.lsmkr).name(),     "checkLockstakeIlkIntegration/incorrect-reg-name");
+            assertEq(symbol,   GemAbstract(p.lsmkr).symbol(),   "checkLockstakeIlkIntegration/incorrect-reg-symbol");
+            assertEq(_class,   7,                               "checkLockstakeIlkIntegration/incorrect-reg-class"); // REG_CLASS_JOINLESS
+            assertEq(decimals, GemAbstract(p.lsmkr).decimals(), "checkLockstakeIlkIntegration/incorrect-reg-decimals");
+            assertEq(gem,      address(mkr),                    "checkLockstakeIlkIntegration/incorrect-reg-gem");
+            assertEq(pip,      p.pip,                           "checkLockstakeIlkIntegration/incorrect-reg-pip");
+            assertEq(gemJoin,  address(0),                      "checkLockstakeIlkIntegration/incorrect-reg-gemJoin");
+            assertEq(clip,     p.clip,                          "checkLockstakeIlkIntegration/incorrect-reg-clip");
+        }
+        // Check required authorizations
+        {
+            assertEq(vat.wards(p.engine),                           1, "checkLockstakeIlkIntegration/missing-auth-vat-engine");
+            assertEq(vat.wards(p.clip),                             1, "checkLockstakeIlkIntegration/missing-auth-vat-clip");
+            assertEq(WardsAbstract(p.pip).wards(address(osmMom)),   1, "checkLockstakeIlkIntegration/missing-auth-pip-osmMom");
+            assertEq(dog.wards(p.clip),                             1, "checkLockstakeIlkIntegration/missing-auth-dog-clip");
+            assertEq(WardsAbstract(p.lsmkr).wards(p.engine),        1, "checkLockstakeIlkIntegration/missing-auth-lsmkr-engine");
+            assertEq(WardsAbstract(p.engine).wards(p.clip),         1, "checkLockstakeIlkIntegration/missing-auth-engine-clip");
+            assertEq(WardsAbstract(p.clip).wards(address(dog)),     1, "checkLockstakeIlkIntegration/missing-auth-clip-dog");
+            assertEq(WardsAbstract(p.clip).wards(address(end)),     1, "checkLockstakeIlkIntegration/missing-auth-clip-end");
+            assertEq(WardsAbstract(p.clip).wards(address(clipMom)), 1, "checkLockstakeIlkIntegration/missing-auth-clip-clipMom");
+        }
+        // Check required OSM buds
+        {
+            assertEq(OsmAbstract(p.pip).bud(address(spotter)), 1, "checkLockstakeIlkIntegration/missing-bud-spotter");
+            assertEq(OsmAbstract(p.pip).bud(p.clip),           1, "checkLockstakeIlkIntegration/missing-bud-clip");
+            assertEq(OsmAbstract(p.pip).bud(address(clipMom)), 1, "checkLockstakeIlkIntegration/missing-bud-clipMom");
+            assertEq(OsmAbstract(p.pip).bud(address(end)),     1, "checkLockstakeIlkIntegration/missing-bud-end");
+        }
+        // Prepare for liquidation
+        uint256 drawAmt;
+        uint256 lockAmt;
+        {
+            // Force max Hole
+            vm.store(address(dog), bytes32(uint256(4)), bytes32(type(uint256).max));
```

Comment:
Nit: use `stdstore` API as below.
```suggestion
            stdstore.target(address(dog)).sig("Hole()").checked_write(bytes32(type(uint256).max));
            assertEq(dog.Hole(), type(uint256).max, "checkLockstakeIlkIntegration/unchanged-Hole");


```

Reply:
I haven't updated this one from `vm.store` to `stdstore` because `stdstore ` is less performant: it basically have to look for the slot in the loop. In this occasion it's a core `dog` contract which we don't expect to change anytime soon. In the [other case](https://github.com/makerdao/spells-mainnet/blob/95c68b5bad2faa03900bd8c973436bf6584fc880/src/DssSpell.t.base.sol#L1337) it's a clipper which exist in two different versions. As it's a nit, and there is one "good to deploy", I would only update it in case there will be more crucial comments in the PR.

Reply:
Fine to leave as it is.